test-case/check-kmod-load-unload.sh

```diff
@@ -110,3 +114,5 @@ do
         sleep 1
     done
 done
+
+disable_devres_debug_log
```

Comment:
I just realized that this will stay enabled after this test when it fails. This needs to be called in the exit hander too.

Reply:
Good point, addressed this issue.